components/site/docsHomePageContent.tsx

```diff
@@ -0,0 +1,374 @@
+ */
+type CardProps = {
+  url: string;
+  title: string;
+  desc: string;
+  linkText: string;
+};
+
+type GuideLinkProps = {
+  text: string;
+  url: string;
+};
+
+type SectionContentProps = {
+  title: string;
+  desc: string;
+  desc2?: JSX.Element;
+};
+
+type LibraryProps = {
+  name: string;
+  url: string;
+  Icon: () => JSX.Element;
+};
+
+type HelpLinkProps = {
+  text: string;
+  url: string;
+  Icon: React.ElementType;
+};
+
+type SimpleStyledLinkProp = {
+  $inlineBlock?: boolean;
+};
+
+/*
+███████╗████████╗██╗   ██╗██╗     ███████╗███████╗
+██╔════╝╚══██╔══╝╚██╗ ██╔╝██║     ██╔════╝██╔════╝
+███████╗   ██║    ╚████╔╝ ██║     █████╗  ███████╗
+╚════██║   ██║     ╚██╔╝  ██║     ██╔══╝  ╚════██║
+███████║   ██║      ██║   ███████╗███████╗███████║
+╚══════╝   ╚═╝      ╚═╝   ╚══════╝╚══════╝╚══════╝
+*/
+
+const SimpleStyledLink = styled.a<SimpleStyledLinkProp>`
+  display: ${(props) => (props.$inlineBlock ? 'inline-block' : 'block')};
+  text-decoration: none;
+  transition: color 0.3s;
+  ${css({
+    marginY: '200',
+  })}
+  &,
+  &:visited {
+    ${css({
+      color: 'gray.900',
+    })}
+  }
+
+  &:hover {
+    ${css({
+      color: 'blue.700',
+    })}
+  }
+
+  ${css({
+    fontWeight: 'normal',
+    textDecoration: 'underline',
+  })}
+`;
+
+const BlueStyledLink = styled.a`
+  text-decoration: none;
+  ${css({
+    paddingY: '200',
+    paddingX: '400',
+    marginRight: '200',
+  })}
+  /* background-color: transparent; */
+  transition: background-color .3s;
+  > svg {
+    ${css({
+      marginRight: '200',
+    })}
+  }
+  &:hover {
+    ${css({
+      backgroundColor: 'blue.200',
+    })}
+  }
+`;
+
+const StyledCard = styled(Box)`
+  &:before {
+    content: '';
+    display: block;
+    position: absolute;
+    top: -1px;
+    left: -1px;
+    right: 0;
+    width: calc(100% + 2px);
+    height: 4px;
+    ${css({
+      backgroundColor: 'blue.700',
+    })}
+  }
+`;
+
+const BoxStyledLink = styled.a`
+  display: flex;
+  align-items: center;
+  width: 30%;
+  text-decoration: none;
+  border-radius: 1px;
+  border-width: 1px;
+  border-style: solid;
+  background-color: transparent;
+  transition: background-color 0.3s;
+  min-width: 165px;
+  ${css({
+    color: 'gray.900',
+    paddingX: '450',
+    paddingY: '300',
+    borderColor: 'gray.900',
+    marginBottom: '500',
+    fontSize: '200',
+    lineHeight: '200',
+  })}
+  &:hover {
+    ${css({
+      backgroundColor: 'gray.300',
+      color: 'gray.900',
+    })}
+  }
+  > svg {
+    ${css({
+      marginRight: '300',
+    })}
+  }
+`;
+
+/*
+ ██████╗ ██████╗ ███╗   ██╗████████╗███████╗███╗   ██╗████████╗
+██╔════╝██╔═══██╗████╗  ██║╚══██╔══╝██╔════╝████╗  ██║╚══██╔══╝
+██║     ██║   ██║██╔██╗ ██║   ██║   █████╗  ██╔██╗ ██║   ██║   
+██║     ██║   ██║██║╚██╗██║   ██║   ██╔══╝  ██║╚██╗██║   ██║   
+╚██████╗╚██████╔╝██║ ╚████║   ██║   ███████╗██║ ╚████║   ██║   
+ ╚═════╝ ╚═════╝ ╚═╝  ╚═══╝   ╚═╝   ╚══════╝╚═╝  ╚═══╝   ╚═╝   
+*/
+
+const gettingStarted = {
```

Comment:
What are your thoughts on moving the content into the `content` directory? maybe something like `content/homePage.ts`?

Reply:
Per our conversation [here in slack](https://msys.slack.com/archives/C02FTBFPNKD/p1637181869012500), this will be a future improvement we'll address later.